doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -1,86 +1,123 @@
-:moveit1:
-
-..
-   Once updated for MoveIt 2, remove all lines above title (including this comment and :moveit1: tag)
-
 Perception Pipeline Tutorial
-============================
+==================================
```

Comment:
When building locally, I get a warning when this does not match the number of heading letters 
```suggestion
============================
```